code/scenarios-streamlined.R

```diff
@@ -93,12 +93,16 @@ desire_lines_combined$length = stplanr::geo_length(desire_lines_combined)
 
 # todo: add PT
 desire_lines_combined = desire_lines_combined %>% 
-  mutate(pwalk_base = foot/all) %>% 
-  mutate(pcycle_base = bicycle/all) %>% 
-  mutate(pdrive_base = car_driver/all)
+  mutate(
+    trimode_base = foot + bicycle + car_driver,
```

Comment:
Reasonable name: it's not all trips, it's trips by the 3 most important modes for our project. Makes me think we should show prominently what the mode share is in the local area. One to add to the `allsites.geojson` dataset, e.g. in columns called `percent_walk`, `percent_drive`, `percent_bus`, `percent_train` `percent_other`. Could also be `percent_walk_2011` more accurately. What do you think @joeytalbot and @mvl22, good idea? can open an issue if it's a :+1: but want to minimise new data/features in the run-up to the workshop.

Reply:
But worth adding the summary data to the `sites-all`/`sites` geojson files? That's more for @mvl22 in terms of how the data is best provided for use by your crazy JS magic, like this

https://github.com/cyipt/actdev/blob/main/data-small/allerton-bywater/site.geojson

(we could add those variables alongside `JobsPTt` etc - I think mode split is a more fundamental and easier to understand metric than the JTS data so my guess is yes)

or like this (note: these files are not present in other files so now is a good time to say if you want them in a different file format @mvl22 ): 

https://github.com/cyipt/actdev/blob/main/data-small/great-kneighton/mode-split.csv

Reply:
I'm sure that will be fine, yeah.

Reply:
> I'm sure that will be fine, yeah.

Yes but which format is easier @mvl22 ?

Reply:
Oh sorry, I misread that. Yes, definitely the site GeoJSON rather than the separate CSV is easier.